Sources/PointFreeTestSupport/TestCase.swift

```diff
@@ -16,7 +16,7 @@ open class TestCase: XCTestCase {
       )
     )
 
-    _ = try! execute("DROP SCHEMA public CASCADE")
+    _ = try! execute("DROP SCHEMA IF EXISTS public CASCADE")
```

Comment:
I canceled running the full suite in just the wrong moment, where `public` had been dropped. Easy fix!

Reply:
oh nice I have also run into this